src/components/WebCLI/Output/Output.tsx

```diff
@@ -110,20 +127,23 @@ const WebCLIOutput = ({
     // New content is coming in, so check if we're collapsed and if we
     // are then open it back up.
     if (
+      // Only trigger this condition if the content changes.
+      content != contentPrevious &&
       content.length > 1 &&
       height <= HELP_HEIGHT &&
       height !== DEFAULT_HEIGHT
     ) {
       setHeight(DEFAULT_HEIGHT);
     }
-    // We can't have height as a dependency because we don't want this to run
-    // when the height changes, only when the content comes in.
-    /* eslint-disable-next-line react-hooks/exhaustive-deps */
-  }, [content]);
+  }, [content, contentPrevious, height]);
```

Comment:
Instead of ignoring these values this now checks whether the content has changed. Without this change the new tests weren't passing.